src/components/Stack.vue

```diff
@@ -335,6 +340,15 @@ export default {
 
               break;
 
+          case 'H':
+            $('button[stackId="'+this.stackId+'"]').attr("data-content", "Hack cards are not played on a stack, you must open the opponent's stack to hack them." );
+            $('button[stackId="'+this.stackId+'"]').popover('toggle');
+            break;
+
+          case 'G':
+            $('button[stackId="'+this.stackId+'"]').attr("data-content", "Group cards are not played on a stack, click the checkbox(s) to group the cards together (the combined value of the stack(s) must equal the value of the group card)." );
```

Comment:
"...stack. Click the checkbox(es) to select the cards to group together. The total of the selected card(s) must equal the value of the group card."